### PR TITLE
229844 Upload Reviewer Step 2 - Return User Uploads data from table storage

### PR DESCRIPTION
**Pull request by gldefra**

### CODE REVIEW

File: EST.MIT.InvoiceImporter.Function/TableEntities/ImportRequestEntity.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public class ImportRequestEntity : ITableEntity
     public string Organisation { get; set; }
     public string SchemeType { get; set; }
     public string AccountType { get; set; }
```
Comment (andygibbonsdefra) [inline EST.MIT.InvoiceImporter.Function/TableEntities/ImportRequestEntity.cs:19]: Are you going to add PaymentType (string)....?

Comment (andygibbonsdefra) [reply EST.MIT.InvoiceImporter.Function/TableEntities/ImportRequestEntity.cs:19]: (And Status (enum)....?)

File: EST.MIT.InvoiceImporter.Function/AutoMapperProfiles/ImportRequestMapper.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace EST.MIT.InvoiceImporter.Function.AutoMapperProfiles;
+
+[ExcludeFromCodeCoverage]
+public class ImportRequestMapper : Profile
+{
+    public ImportRequestMapper()
+    {
+        CreateMap<ImportRequestEntity, ImportRequest>();
+
```
Comment (andygibbonsdefra) [inline EST.MIT.InvoiceImporter.Function/AutoMapperProfiles/ImportRequestMapper.cs]: Whitespace.

**Review by jbarnsley10 — APPROVED** [review]
Looks great Gavin. The only thing I'd suggest adding (and you don't have to do this) is a unit test that does 'configuration.AssertConfigurationIsValid();' to prove the auto mapper config. You could do that in your Startup class too but as our pipelines enforce unit tests, I think a unit test is better. It's trivial in this example where your model and entity map easily but if someone adds more automapper profiles, this is worth adding.
I also noticed a .DS_Store file that had been checked in (not in your PR) but we should probably add that to the .gitignore so Mac users don;t end up checking in loads of .DS_Store files

**Review by andygibbonsdefra — APPROVED** [review]

**Pull request merged**